scripts/start-relayer.sh

```diff
@@ -1,33 +1,48 @@
 #!/bin/bash
 
-RELAY_PATH="${RELAY_PATH:-local_testnet}"
+RELAY_PATH="${RELAY_PATH:=local_local2}"
```

Comment:
Is this change from `local_testnet` to `local_local2` intended?

Reply:
It was originally intended to be local_testing to easily connect a local blockchain to testnet, but that's impossible to do out of the box unless they rename their chain id. I thought it might be easier if they have two local blockchains, but that also requires some setup. It also may not be possible for everyone to do that. I'll change it back to local_testnet.